Sources/Models/SubscribeConfirmationData.swift

```diff
@@ -3,13 +3,16 @@ import Stripe
 
 public struct SubscribeConfirmationData: Equatable {
   public var billing: Pricing.Billing
+  public var isOwnerTakingSeat: Bool
```

Comment:
this is to allow restoring this piece of state when we redirect back to the subscription confirmation page in case of a stripe error.